app/lib/frontend/templates/views/shared/listing_info.dart

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) 2021, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import '../../../../shared/markdown.dart';
+import '../../../dom/dom.dart' as d;
+
+d.Node renderListingInfoNode({
+  required int totalCount,
+  required String? searchQuery,
+  required String? ownedBy,
+  required d.Node sortControlNode,
+  required String? messageMarkdown,
+}) {
+  final packageOrPackages = totalCount == 1 ? 'package' : 'packages';
+  return d.div(
+    classes: ['listing-info'],
+    children: [
+      d.div(
+        classes: ['listing-info-count'],
+        children: [
+          d.span(classes: ['info-identifier'], children: [d.text('Results')]),
+          d.span(classes: ['count'], children: [d.text('$totalCount')]),
+          d.text(' $packageOrPackages'),
+          if (searchQuery != null && searchQuery.isNotEmpty) ...[
+            d.text(' for search query '),
+            d.code(children: [d.text(searchQuery)]),
+          ],
+          if (ownedBy != null && ownedBy.isNotEmpty) ...[
+            d.text(' owned by '),
+            d.code(children: [d.text(ownedBy)]),
+          ],
+          if (messageMarkdown != null)
+            d.unsafeRawHtml(markdownToHtml(messageMarkdown)!),
```

Comment:
May I propose that we make a `d.markdown` method?

This way, the `d.unsafeRawHtml` will only be used in a very few places, and it'll be easy to review.